docs/content/en/_index.md

```diff
@@ -48,47 +48,51 @@ configure:
   filename: "/etc/vector/vector.toml"
   below: "Configuration examples are in [TOML](https://toml.io) but Vector also supports [YAML](https://yaml.org) and [JSON](https://json.org)"
   example_configs:
-  - title: Parse NGINX logs
+  - title: Redacted Datadog Agent logs to Datadog
     config: |
-      [sources.nginx_error_log]
-      type = "file"
-      include = ["/var/log/nginx*.log"]
-      start_at_beginning = false
-      ignore_older = 86400
+      [sources.datadog_agent]
+      type = "datadog_logs"
+      address = "localhost:80"
 
-      [transforms.nginx_error_parser]
-      inputs = ["nginx_error_log"]
+      [transforms.remove_sensitive_user_info]
       type = "remap"
+      inputs = ["datadog_agent"]
       source = '''
-      # TODO
+        redact(., filters: ["us_social_security_number"])
       '''
-  - title: Something else
-    config: |
-      [sources.do_something_else]
-      type = "other"
 
-      [sinks.out]
-      inputs = ["in"]
-      type = "console"
-      encoding.codec = "text"
-  - title: Other thing entirely
+      [sinks.datadog_backend]
+      type = "datadog_logs"
+      inputs = ["remove_sensitive_info"]
+      default_api_key = "${DATADOG_API_KEY}"
+
+  - title: Kafka topic to Elasticsearch
     config: |
-      [sources.do_other_thing_entirely]
-      type = "other"
+      [sources.kafka_in]
+      type = "kafka"
+      bootstrap_servers = "10.14.22.123:9092,10.14.23.332:9092"
+      group_id = "vector-logs"
+      message_key = "message"
+      topics = ["logs-*"]
+
+      [sinks.elasticsearch_out]
+      type = "elasticsearch"
+      inputs = ["kafka_in"]
```

Comment:
Technically I think we need to `parse_json` before sending across to ES

Reply:
Okay, I updated this. Let me know if the new version works. I'm a bit rusty with configs.